recipes-devtools/rust/rust-version.inc

```diff
@@ -1,6 +1,6 @@
 # Note: if you adjust this, you'll also need to change the hashes in
 # rust-source.bb
 SOURCE_NAME = "rust"
-PV = "1.10.0"
+PV = "1.12.0"
```

Comment:
Please note that 1.12.1 has been released and is probably the one you want to be targeting.